src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,104 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+
 public class WorkWithFile {
+    static final int MAX_LINES = 100;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] rawFields = readToTable(fromFileName);
+        String[] sumFields = sumTogether(rawFields);
+        saveToFile(sumFields, toFileName);
+    }
+
+    public String [] readToTable(String fileName) {
+        String [] temporaryFields = new String[MAX_LINES];
+        int count = 0;
+        File file = new File(fileName);
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+
+            while (value != null) {
+                temporaryFields[count] = value;
+                count++;
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read", e);
```

Comment:
```suggestion
            throw new RuntimeException("Cannot read data from file " + fileName, e);
```

lets make exception messages more informative